### PR TITLE
expand ruler ring section to add num_tokens property

### PR DESCRIPTION
**Pull request by trevorwhitney**
**What this PR does / why we need it**:

Updates the ring's ruler's configuration section in the docs. The ruler ring was previously using the shared `ring_config`, but that does not expose `num_tokens`, which can be configured for the ruler's ring

**Checklist**
- [ ] Documentation added
- [ ] Tests updated
- [ ] Add an entry in the `CHANGELOG.md` about the changes.

### CODE REVIEW

File: docs/sources/configuration/_index.md [hunk 1]
```diff
@@ -599,6 +599,61 @@ remote_write:
...
+      # CLI flag: -<prefix>.multi.mirror-timeout
+      [mirror_timeout: <duration> | default = 2s]
+
+  # Interval between heartbeats sent to the ring. 0 = disabled.
+  # CLI flag: -<prefix>.heartbeat-period
+  [heartbeat_period: <duration> | default = 15s]
+
+  # The heartbeat timeout after which store gateways are considered unhealthy
+  # within the ring. 0 = never (timeout disabled). This option needs be set both
+  # on the store-gateway and querier when running in microservices mode.
```
Comment (KMiller-Grafana) [inline docs/sources/configuration/_index.md:644]: Should this description be modified?  It seems to talk about gateways and queriers, and this description is in the section on rulers.

**Review by kavirajk — APPROVED** [review]
LGTM 👍

File: docs/sources/configuration/_index.md (hunk 1 above)
Comment (cyriltovena) [reply docs/sources/configuration/_index.md:644]: I think so too.

Comment (trevorwhitney) [reply docs/sources/configuration/_index.md]: done

**Pull request merged**